1-exercises/A-reading-list/script.js

```diff
@@ -1,7 +1,33 @@
 function readingList(books) {
```

Comment:
I wonder what this function does.  Well actually I know it was given by CYF for you to fill out.  Naming can be hard sometimes but when well done it helps other members of the team who may later work on the same code. Or you when you come back to code after some time.